deploy.sh

```diff
@@ -84,62 +147,99 @@ AKS_NAME=`echo ${BINDERHUB_NAME} | tr -cd '[:alnum:]-' | cut -c 1-59`-AKS
 
 
 if [ -z $SP_APP_ID ] || [ -z $SP_APP_KEY ] || [ -z $SP_TENANT_ID ] ; then
-  echo "Attempting to log in to Azure as a user"
+  echo "--> Attempting to log in to Azure as a user"
   if ! az login -o none; then
-      echo "Unable to connect to Azure" >&2
+      echo "--> Unable to connect to Azure" >&2
       exit 1
+  else
+      echo "--> Logged in to Azure"
   fi
 else
-  echo "Attempting to log in to Azure with service principal"
+  echo "--> Attempting to log in to Azure with service principal ${SP_APP_ID}"
```

Comment:
I'm not 100% sure we should be echoing the SP appID here. It's not really the kind of thing you want lingering in logs and/or terminal history.

Reply:
I'd be happier with:
```
SP_NAME=`az ad sp show --id ${SP_APP_ID} --query displayName -o tsv`
```
and echoing that.

Reply:
Fixed! https://github.com/alan-turing-institute/binderhub-deploy/pull/22/commits/64a5d159f06e8628eb95b6cccd80a619823f7361